app/javascript/elm/src/Main.elm

```diff
@@ -448,10 +448,10 @@ toGraphParams thresholds selectedSession sensors selectedSensorId =
             HeatMapThresholds.toValues thresholds
 
         levels =
-            [ { from = threshold1, to = threshold2, color = "#aaa" }
-            , { from = threshold2, to = threshold3, color = "#bbb" }
-            , { from = threshold3, to = threshold4, color = "#ccc" }
-            , { from = threshold4, to = threshold5, color = "#ddd" }
+            [ { from = threshold1, to = threshold2, className = "first-band" }
+            , { from = threshold2, to = threshold3, className = "second-band" }
+            , { from = threshold3, to = threshold4, className = "third-band" }
+            , { from = threshold4, to = threshold5, className = "fourth-band" }
```

Comment:
why `-band`? (just trying to understand the meaning, what it reflects)

Reply:
(in case I wanna change sth here as you suggested)

Reply:
another words: what's a graph band?

Reply:
I guess this should answer the question https://www.highcharts.com/docs/chart-concepts/plot-bands-and-plot-lines